openhands/server/session/agent_session.py

```diff
@@ -425,7 +439,9 @@ async def _create_memory(
 
         if self.runtime:
             # sets available hosts and other runtime info
-            memory.set_runtime_info(self.runtime, custom_secrets_descriptions)
+            memory.set_runtime_info(
```

Comment:
@openhands change the name of memory.set_runtime_info to memory.set_contextual_info